sound/soc/sof/sof-priv.h

```diff
@@ -204,6 +204,8 @@ struct snd_sof_dsp_ops {
 	int (*load_firmware)(struct snd_sof_dev *sof_dev); /* mandatory */
 	int (*load_module)(struct snd_sof_dev *sof_dev,
 			   struct snd_sof_mod_hdr *hdr); /* optional */
+	int (*load_library)(struct snd_sof_dev *sdev,
+			    struct snd_sof_module_library_info *lib_info); /* optional */
```

Comment:
Are we support or going to support external libraries with IPC3 or other IPC versions?
Should this be a private IPC4 thing?